static/app/views/dashboards/widgetBuilder/components/widgetTemplatesList.tsx

```diff
@@ -0,0 +1,111 @@
+import {useState} from 'react';
+import styled from '@emotion/styled';
+
+import {Button} from 'sentry/components/button';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import theme from 'sentry/utils/theme';
+import useOrganization from 'sentry/utils/useOrganization';
+import {getTopNConvertedDefaultWidgets} from 'sentry/views/dashboards/widgetLibrary/data';
+import {getWidgetIcon} from 'sentry/views/dashboards/widgetLibrary/widgetCard';
+
+function WidgetTemplatesList() {
+  const organization = useOrganization();
+  const [selectedWidget, setSelectedWidget] = useState<number | null>(null);
+
+  const widgets = getTopNConvertedDefaultWidgets(organization);
+
+  return (
+    <div>
+      {widgets.map((widget, index) => {
+        const iconColor = theme.charts.getColorPalette(widgets.length - 2)?.[index]!;
+
+        const Icon = getWidgetIcon(widget.displayType);
+        const lastWidget = index === widgets.length - 1;
+        return (
+          <TemplateContainer key={widget.id} lastWidget={lastWidget}>
+            <TemplateCard
+              selected={selectedWidget === index}
+              onClick={() => setSelectedWidget(index)}
+            >
+              <IconWrapper backgroundColor={iconColor}>
+                <Icon color="white" />
+              </IconWrapper>
+              <ContentWrapper>
```

Comment:
If this doesn't need any styling, I would just replace it with a `div`, but if you're keeping it around for something later, then ignore me 👍